frame/election-provider-support/solution-type/src/single_page.rs

```diff
@@ -102,7 +102,7 @@ pub(crate) fn generate(def: crate::SolutionDef) -> Result<TokenStream2> {
 		#vis struct #ident { #single #rest }
 
 		use _npos::__OrInvalidIndex;
```

Comment:
is this the only pace where this _npos is used?

Reply:
I removed _npos from everywhere and replaced it with _fpes (not sure if this name is ok as a shorthand for `frame_election_provider_support`?)